tests/boost/test_ringbuffer.cpp

```diff
@@ -18,6 +19,23 @@ using json = nlohmann::json;
 using namespace kotekan;
 using namespace std;
 
+// Use a boost::test "Global Fixture" to set the locale for fmt...
+// https://www.boost.org/doc/libs/1_75_0/libs/test/doc/html/boost_test/tests_organization/fixtures/global.html
+struct GlobalFixture_Locale {
```

Comment:
Should this be moved into `testUtils.hpp` since I assume more than one test might want to make use of it?

Reply:
Good idea, done!